File: ОП/Lab2/Lab2 C++/laboratorna2/laboratorna2.cpp
#include <iostream>
#include <math.h>
using namespace std;
int main()

{
	double x;
	double y;

	cout << "Type in x:";
	cin >> x;

	cout << "Type in y:";
	cin >> y;

	if (pow(x-1, 2) + pow(y, 2) <= 4 && -1 <= x && x <= 1) {
		cout << "Point (x,y) belongs to figure";
	}

	else if (abs(y) <= 3-x  && 1 <= x && x <= 3) {
		cout << "Point (x,y) belongs to figure";
	}

	else { cout << "Point (x,y) DOESN`T belong to figure"; }

}

File: ОП/Lab2/Lab2 Python/laboratorna2.py
x=float(input("Введіть x: "))
y=float(input("Введіть y: "))
if (x-1)**2+y**2<=4 and -1<=x and x<=1:
    print("Point (x,y) belongs to figure")
elif abs(y)<=3-x and 1<=x and x<=3:
    print("Point (x,y) belongs to figure")    
else:
    print("Point (x,y) DOESN`T belong to figure")